docs/intro/contributing.txt

```diff
@@ -603,23 +603,53 @@ This patch file contains all your changes and should look this:
              # NullBooleanField is a bit of a special case because its presentation (widget)
              # is different than its data. This is handled transparently, though.
 
-So what do I do next?
-=====================
+Once you're done looking at the patch, hit the ``q`` key to exit back to the
+command line. If the patch's content looked okay, it is time to commit all the
+changes.
 
-Congratulations, you've generated your very first Django patch! Now that you've
-got that under your belt, you can put those skills to good use by helping to
-improve Django's codebase. Generating patches and attaching them to Trac
-tickets is useful, however, since we are using git - adopting a more :doc:`git
-oriented workflow </internals/contributing/writing-code/working-with-git>` is
-recommended.
+Committing the changes in the patch
+===================================
 
-Since we never committed our changes locally, perform the following to get your
-git branch back to a good starting point:
+To commit the changes, use the following command
 
 .. code-block:: console
 
-    $ git reset --hard HEAD
-    $ git checkout master
+    $ git add django/forms/forms.py docs/ref/forms/api.txt docs/releases/1.9.txt 
+      tests/forms_tests/ tests/test_forms.py
+    $ git commit 
+
+This opens up a text editor to type the commit message. Do remember to follow
+the `committing guidelines`__ while writing a commit message.
+
+__ https://docs.djangoproject.com/en/dev/internals/contributing/committing-code/#committing-guidelines
+
+Pushing the commit and making a Pull Request
+============================================
+
+After committing the patch, it has to updated on the fork on GitHub too.
+You the following command to push your commits
+
+.. code-block:: console
+
+    $ git push origin branchName
+
+
+You can create a pull request by visiting `pull request`__ link.
```

Comment:
I usually create a pull request using https://github.com/django/django and the "Your recently pushed branches:" are that appears. I think that's a bit simpler, is it okay with you?